src/components/toolbar/toolbar.js

```diff
@@ -77,13 +73,12 @@ const Toolbar = ({
    * Set selected options for chip display.
    *
    * @param {object} params
-   * @param {Array} params.options
    * @param {*} params.value
    * @returns {Array}
    */
-  const setSelectedOptions = ({ options: filterOptions, value }) => {
-    const query = filterOptions.find(({ value: optionValue }) => optionValue === toolbarFieldQueries?.[value]);
-    return (query?.title && [query?.title]) || [];
+  const setSelectedOptions = ({ value } = {}) => {
+    const categoryValue = toolbarFieldQueries?.[value];
+    return (categoryValue && [t('curiosity-toolbar.label', { context: [value, categoryValue] })]) || [];
```

Comment:
required:
testing caught the "truthy" check when a query param is purposefully set to  empty, `""`. maybe do a `typeof` `string` check?